src/agent/solana/exporter.rs

```diff
@@ -88,6 +87,7 @@ const UPDATE_PRICE_NO_FAIL_ON_ERROR: i32 = 13;
 // measure while using dynamic compute price to prevent the exporter from paying too much for a
 // single transaction
 const MAXIMUM_TOTAL_COMPUTE_UNIT_FEE_MICRO_LAMPORTS: u64 = 1_000_000_000_000;
+const MAXIMUM_SLOT_GAP_FOR_DYNAMIC_COMPUTE_UNIT_PRICE: u64 = 25;
```

Comment:
I suggest putting both of these consts into the config. these are the kind of thing that we'll probably want to tune easily in an emergency.